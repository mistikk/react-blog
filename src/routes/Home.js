import React from 'react';
import { Link } from 'react-router-dom';

const Home = () =>
  <div>
    <h2>Ana Sayfa</h2>
    <Link to="/basla">Denemeye Basla</Link>
  </div>;

export default Home;
