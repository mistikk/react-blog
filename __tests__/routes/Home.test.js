import React from 'react';
import { shallow } from 'enzyme';

import Home from '../../src/routes/Home';

it('renders correctly', () => {
  const wrapper = shallow(<Home />);

  expect(wrapper).toMatchSnapshot();
});
