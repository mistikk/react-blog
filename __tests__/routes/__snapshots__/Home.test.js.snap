exports[`test renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <h2>
        Ana Sayfa
    </h2>
    <Link
        replace={false}
        to="/basla">
        Denemeye Basla
    </Link>
</div>,
  "nodes": Array [
    <div>
      <h2>
            Ana Sayfa
      </h2>
      <Link
            replace={false}
            to="/basla">
            Denemeye Basla
      </Link>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Home />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <h2>
                    Ana Sayfa
          </h2>
          <Link
                    replace={false}
                    to="/basla">
                    Denemeye Basla
          </Link>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <h2>
                    Ana Sayfa
          </h2>
          <Link
                    replace={false}
                    to="/basla">
                    Denemeye Basla
          </Link>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Home />,
}
`;
